robot-name/robot-name_test.spec.js

```diff
@@ -12,9 +12,17 @@ describe("Robot", function() {
   });
 
   it("different robots have different names", function() {
-    var robotOne = new Robot();
-    var robotTwo = new Robot();
-    expect(robotOne.name).not.toEqual(robotTwo.name);
+    var usedNames= {};
+
+    var i = 0,
+        max = 10000;
+
+    for (i; i < max; i++) {
```

Comment:
Wow, ok that does make sense. I still like it better with `for (var i = 0; i < 1000; i++)`, to be honest.

I'm not a JavaScript dev, though, so if the two of you can agree on something, I'll leave the final decision to you :)